### PR TITLE
[PLATFORM-1007] Renamed MostimagesInContentPage to MostLinkedFilesInContentPage

### PR DESCRIPTION
**Pull request by jcellary**
https://wikia-inc.atlassian.net/browse/PLATFORM-1007

Renamed special page MostimagesInContentPage to MostLinkedFilesInContentPage created earlier as requested by community support.

The actual class has been temporarily duplicated due to caching of query result once every 24 hours. So the classes will run in parallel for a few days and then I will delete the old unneeded class.

I'm not experienced in making changes in this repo, so please take a good look at my PR ;)

Pinging @macbre @michalroszka @wladekb

### CODE REVIEW

File: extensions/wikia/ImageServing/drivers/ImageServingDriverMainNS.class.php [hunk 1]
```diff
@@ -189,19 +189,19 @@ protected function getImagesPopularity( Array $imageNames, $limit ) {
...
 
-		// MostimagesInContentPage generates daily reports with images
+		// MostLinkedFilesInContentPage generates daily reports with images
 		// that are included at least twice in article in content namespaces
 		$ret = $this->db->select(
 			'querycache',
 			[ 'qc_title as image', 'qc_value as popularity' ],
 			[
-				'qc_type' => 'MostimagesInContent',
+				'qc_type' => 'MostLinkedFilesInContent',
```
Comment (macbre) [inline extensions/wikia/ImageServing/drivers/ImageServingDriverMainNS.class.php]: After the release there will be no data with `'qc_type' = 'MostLinkedFilesInContent'`. Cron job regenerates this data via `rebuildSpecialPages` script every day.

**Comment by macbre** [comment]
`updateSpecialPages` fails for me:

```
macbre@dev-macbre:~/app/maintenance$ SERVER_DBNAME=plpoznan php updateSpecialPages.php 

MostLinkedFilesInContent       got 309 rows in 0.05s
No such special page: MostimagesInContent
```

**Comment by jcellary** [comment]
@macbre I removed the old class as discussed offline. I suggest merging it end of this week, to minimaze the query update gap

**Comment by wladekb** [comment]
how about leaving the old name as ana alias to the new one?

**Comment by jcellary** [comment]
We did something similar at first, but it lead to problems, let's talk offline about it

**Comment by macbre** [comment]
The problem here is that `wgQueryPages` relies on class names, not special pages names.

**Comment by wladekb** [comment]
Maybe we can create a fake special page under the previous name that just redirects to the new one (along with all the parameters passed)?

**Comment by jcellary** [comment]
Ok Macbre created a separate PR for the new approach, so closing this one.

**Pull request closed without merging**